packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/types/AsymmetricRealmObject.kt

```diff
@@ -0,0 +1,25 @@
+package io.realm.kotlin.mongodb.types
```

Comment:
Yeah, I was having the same debate with myself and there are advantages to both. I kinda like that the package implies this is sync only, but we don't really have any other types lined up, and if we end up combining the sync and base libraries it will probably also be annoying.

Hmm...I think I'm talking my way into moving it into `io.realm.kotlin.types` after all.